packages/quantic/docs/jsdoc-guidelines.md

```diff
@@ -0,0 +1,72 @@
+# JSDoc Guidelines for Quantic Components
```

Comment:
@jpmarceau what do you think of this?

Reply:
Could we have the component `categories` display as badges in the component's documentation?

Reply:
I believe it would be possible to add badges, but I don't know if it's useful considering what I've implemented:
https://docsstaging.coveo.com/quantic-categories-demo/quantic/latest/reference/
The TOC simply reflects the category structure :) 
Do you think that's enough or we'd like a badge too?

Reply:
BTW, I just realized by going through this Readme that components can be in multiple categories, and my script doesn't currently handle that, so you may see missing pages in my staging link until I fix that. (I'm missing pages for components that show up under multiple categories)

Reply:
Yeah i hadnt alerted you about that yet. That's the idea behind the badges. So we can support components being part of more than one category. Right now none of them are though.
The idea is not to limit ourselves with this new structure while still providing a way to view a relevant subset of components. Components with multiple categories would show up in multiple places. What do you think about that?

Reply:
Yeah I can adjust my script so components show up in many categories when needed (that's the case of the ResultLink in fact).
I just don't really see the value of displaying badges if the components already show up in the target places in the TOC 🤔

Reply:
The idea behind the badges is 
1) to display all component metadata on the component page, which includes categories
2) so users aren't confused when they see it show up multiple places. I think the badges would make it more obvious that it's the same component

Reply:
Ah yeah I see that makes sense 👍 
I'll implement something like what we do in headless for controllers that show up under several engines:
https://docs.coveo.com/en/headless/latest/reference/controllers/did-you-mean/#associated-engines